### PR TITLE
[Web2Print] Render twig templates in a sandbox

### PR DESCRIPTION
**Pull request by dvesh3**
## Changes in this pull request  
follow up to #13347

### CODE REVIEW

File: doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Upgrade Notes
 
+## 10.5.9
```
Comment (jdreesen) [inline doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md]: `10.5.9` is already released.

```suggestion
## 10.5.10
```

Comment (dvesh3) [reply doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md]: Oops, we also have to fix the last upgrade note.

**Review by fashxp — DISMISSED** [review]
we need to discuss that in more detail before merging

**Comment by dvesh3** [comment]
As discussed, this change is still required as the `processHtml()` should render the twig expressions (pre-rendered from `renderDocument()` in `buildPdf`) in a sandbox, just for the purpose of replacing the placeholders with parameters.

File: doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 # Upgrade Notes
 
+## 10.5.10
```
Comment (dvesh3) [inline doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md:6]:
```suggestion
## 10.5.13
```

File: doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md [hunk 3]
```diff
@@ -1,5 +1,8 @@
 # Upgrade Notes
 
+## 10.5.13
+- [Web2Print] Print document twig templates are now executed in a sandbox with restrictive security policies (just like Sending mails and Dataobject Text Layouts introduced in 10.5.8).
```
Comment (dvesh3) [inline doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md]:
```suggestion
- [Web2Print] Print document twig expressions are now executed in a sandbox with restrictive security policies (just like Sending mails and Dataobject Text Layouts introduced in 10.5.8).
```

File: doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md [hunk 4]
```diff
@@ -1,5 +1,8 @@
 # Upgrade Notes
 
+## 10.5.13
+- [Web2Print] Print document twig expressions are now executed in a sandbox with restrictive security policies (just like Sending mails and Dataobject Text Layouts introduced in 10.5.8).
```
Comment (blankse) [inline doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md]:
```suggestion
- [Web2Print] Print document twig expressions are now executed in a sandbox with restrictive security policies (just like Sending mails and Dataobject Text Layouts introduced in 10.5.9).
```

**Pull request merged**